CHANGELOG.md

```diff
@@ -11,6 +11,10 @@
 - utils `v0.0.23`
 - nns-proto `v0.0.9`
 
+## Breaking Changes ⚠️
```

Comment:
> Shouldn't we copy the code first and set as deprecated the one in @dfinity/nns until we have a new major release as you mention

Not sure JavaScript supports a deprecated tags. That's an option but my idea is rather that we release a v1 to finalize the existing and release afterwards a v2 with the breaking changes.

Reply:
> How about we import and export it exactly the same in nns-js during some time?

Not sure I understand what you mean?

Reply:
That nns-js depends on ledger-icp. It imports the LedgerCanister and types from it and exports them.

But maybe too much...

Reply:
Do I get it right that you are suggesting to re-export the `LedgerCanister` types within `@dfinity/nns`?

That would be incorrect in my opinion because then consumers have `import Ledger from nns` and `import Ledger from ledger-icp`. Furthermore, given the fact that ledger-icp is a peer dependency of nns, that particular library is present on the consumer side.

But maybe I still do not understand your suggestion.